### DISCUSSION TITLE
Information about automic package updating

### DISCUSSION THREAD

**Discussion by michha** [chocolatey-community/chocolatey-packages#2451] [Q&A] [answered]
Where can I find information about the automatic update process?

The cause of this question is the [nodejs package](https://community.chocolatey.org/packages/nodejs).
Security patches were published yesterday and the package is not updated yet.

I want to know more about the updating process. How is it triggered? Has someone be notified about a new version?
Is this true for automatic packages? Or do I need to file an issue for this.

What I found until now is:
- [Package Triage Process](https://docs.chocolatey.org/en-us/community-repository/users/package-triage-process)
- [NodeJS package files](https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/nodejs)
- [Wiki](https://github.com/chocolatey-community/chocolatey-packages/wiki) (first two links are dead 😢)
- the commit history of the automatic packages without a clue how these are triggered
![image](https://github.com/chocolatey-community/chocolatey-packages/assets/10424767/0809108d-3a21-4800-b2e3-34666db9dc04)

**Comment by AdmiringWorm** [comment, marked as answer]
The automatic updates in this repository is using a module called [`Chocolatey-AU`](https://github.com/chocolatey-community/chocolatey-au), and is triggered as part of a cron job on AppVeyor (I forget how often, but several times a day). We can trigger a forced update of a package, but this is generally not done.

Notifications about failures is sent to both common (private) emailing list, and is updated as part of the gist here: https://gist.github.com/choco-bot/a14b1e5bfaf70839b338eb1ab7f8226f

In the terms of nodejs explicitly, it looks like it is failing due to getting the same binary for both 32bit and 64bit installers (and to prevent incompatible packages, it isn't updated in this case).

> Or do I need to file an issue for this.

As this, this is a community maintained repository, the best option, no matter the situation, is to file an issue about the package being outdated (only for stable releases), or optionally provide a PR to address any issue that causes the package to not be updated.

**Reply by AdmiringWorm** [reply]
> [Wiki](https://github.com/chocolatey-community/chocolatey-packages/wiki) (first two links are dead 😢)

I have updated those two links now to point to the correct location, thanks for bringing those to our attention.

**Reply by michha** [reply]
Is the gist link somewhere linked? I couldnt find it.

**Reply by AdmiringWorm** [reply]
It is yes. For the [chocolatey-packages](https://github.com/chocolatey-community/chocolatey-packages/blob/master/README.md) repository itself, it is listed in the Readme.md file in that repository (for other repositories, it can be different).

There is a link called `Update Status` that links to that gist.

**Reply by michha** [reply]
Thanks for your answers and your time 😄
